plugins/catalog-backend-module-gcp/src/providers/GkeEntityProvider.ts

```diff
@@ -142,8 +154,14 @@ export class GkeEntityProvider implements EntityProvider {
             [ANNOTATION_KUBERNETES_API_SERVER_CA]:
               cluster.masterAuth?.clusterCaCertificate || '',
             [ANNOTATION_KUBERNETES_AUTH_PROVIDER]: 'google',
-            'backstage.io/managed-by-location': location,
-            'backstage.io/managed-by-origin-location': location,
+            [ANNOTATION_KUBERNETES_DASHBOARD_APP]: 'gke',
+            [ANNOTATION_LOCATION]: location,
+            [ANNOTATION_ORIGIN_LOCATION]: location,
+            [ANNOTATION_KUBERNETES_DASHBOARD_PARAMETERS]: JSON.stringify({
```

Comment:
couldn't this just be a `metadata.link` to the dashboard, rather than the parameters of a dashboard?

Storing a json string in an annotation feels a bit strange

Reply:
Having it as parameters allows link customisation at runtime to e.g. point to particular deployment.
https://github.com/backstage/backstage/blob/master/plugins/kubernetes-react/src/utils/clusterLinks/formatters/gke.ts
Also storing JSON as annotation field is a [standard practice](https://kubernetes.io/docs/reference/labels-annotations-taints/#schedulerkubernetestolerations-whitelist)

Reply:
Also, because dashboardParameters can hold arbitrary values (to support different cloud providers) it's not really possible to decompose it to multiple annotations.

Reply:
I see. Very well.

Reply:
I wonder if there's any way to "type" the parameters for the API report, as a JSON string can be anything really

Reply:
From what I saw they are JsonObject in TS classes.
It also makes sense, as any cluster provider might need a different one.
Also users can create custom providers with wildly different types.